host/class/uvc/usb_host_uvc_2/host_test/main/parsing/descriptors/old.hpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+// Descriptors from UVC driver version 1
+// TODO check and test this
```

Comment:
Yes, I did not add this testing todo to the list. The descriptor parsing has a pretty good coverage and I don't have this cameras so it is not implemented.

It can be removed, I wanted to leave it here for future reference